helm-chart/helm2/6.0.0-GA/skywalking/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 appVersion: 6.0.0-GA
 description: Apache SkyWalking APM System
 version: 0.1.0
-icon: https://github.com/apache/skywalking-kubernetes/blob/master/logo/skywalking-logo2.png
+icon: https://raw.githubusercontent.com/apache/skywalking-kubernetes/master/logo/sw-logo-for-chart.jpg
```

Comment:
Why does this URL change?

Reply:
Pushed to chartmuseum .  Logo does not display properly